src/esp/gfx/GenericDrawable.h

```diff
@@ -40,9 +40,6 @@ class GenericDrawable : public Drawable {
   Magnum::ResourceKey getShaderKey(Magnum::UnsignedInt lightCount,
                                    Magnum::Shaders::Phong::Flags flags) const;
 
-  Magnum::GL::Texture2D* texture_;
-  Magnum::Color4 color_;
```

Comment:
Huh. Just to be clear -- those are stored in the `materialData_` now, right?

Reply:
@mosra: Yes, I think so.